scrapers/utils/secrets.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import json
+
+
+def get_secret(secret_name, region="us-west-2"):
+    """
+    Retrieves secret value from a secret in AWS Secrets Manager.
+
+    Parameters:
+    - secret_name (str): The name of the secret.
+    - region (str): The AWS region where the secret is stored.
+
+    Returns:
+    - The value associated with the specified key.
+    """
+
+    client = boto3.client(service_name="secretsmanager", region_name=region)
+
+    try:
+        secret_response = client.get_secret_value(SecretId=secret_name)
+
+    except Exception as e:
+        print(f"Error retrieving secret: {e}")
```

Comment:
is there no existing logger?

Reply:
It looks like the openstates-core and openstates-scrapers repos use the `logging` library - if it makes sense to you, I can remove the print statements and add in `logging` warnings instead!

Reply:
UPDATE: I added in `logging` warnings in lieu of print statements.